.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Enhancement
+about: Suggest an idea for this project
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+# Context
+
+(Is your feature request related to a problem?)  
```

Comment:
```suggestion
(Describe what you want to change,  and why this is a good idea)  
```

Reply:
Did it in https://github.com/inmanta/inmanta/pull/2416/commits/2e2f1f6a155d159d30daa2953c86835f4bde579f